sinks/api/supported_labels.go

```diff
@@ -15,13 +15,15 @@
 package api
 
 const (
-	LabelPodId         = "pod_id"
-	LabelPodName       = "pod_name"
-	LabelPodNamespace  = "pod_namespace"
-	LabelContainerName = "container_name"
-	LabelLabels        = "labels"
-	LabelHostname      = "hostname"
-	LabelResourceID    = "resource_id"
+	LabelPodId               = "pod_id"
+	LabelPodName             = "pod_name"
+	LabelPodNamespace        = "pod_namespace"
+	LabelContainerName       = "container_name"
+	LabelLabels              = "labels"
+	LabelHostname            = "hostname"
+	LabelResourceID          = "resource_id"
+	LabelComputeResourceID   = "compute.googleapis.com/resource_id"
```

Comment:
I can do it, but there will be name collision. There is already label named "resource_id". The solution is to change the value of LabelResourceID. WDYT?